src/code/z_message.c

```diff
@@ -1,13 +1,12 @@
 #include "z64message.h"
-#include "message_data_static.h"
+#include "global.h"
```

Comment:
How come `"global.h"` is reintroduced here?

Reply:
Because of `z_kaleido_scope.h`. That header was including `global.h`, so removing `z_kaleido_scope.h` caused missing declarations errors.